src/pds_doi_service/api/swagger/swagger.yaml

```diff
@@ -125,6 +125,8 @@ paths:
         for one or more  records to be submitted. Record URLs must resolve to PDS4
         label files (xml).
       operationId: post_dois
+      security:
+      - jwt: ['']
```

Comment:
When I see a colon on a line above, I reflexively want to see an indentation on the next line:
```yaml
security:
  - jwt: ['']
```
Although `yamllint` says they evaluate to the same data structure. (Perhaps I program too much Python? 😁)

Reply:
@nutjob4life, the Swagger formatting in the above line 128 and 129 is consistent with the format of rest of the content in the file and also with YAML files generated by AWS API Gateway etc. So, I think it is ok to keep it as it is.